README.md

```diff
@@ -16,17 +16,63 @@
 <h3 align="center">
   <a href="https://demo.nussknacker.io"><b>Demo</b></a> &bull;
   <a href="https://nussknacker.io/documentation"><b>Documentation</b></a> &bull;
+  <a href="https://nussknacker.io/documentation/quickstart/lite-streaming/"><b>Quickstart</b></a> &bull;
   <a href="https://nussknacker.io/documentation/docs/installation/"><b>Installation</b></a> &bull;
   <a href="https://cloud.nussknacker.io"><b>Nu Cloud</b></a>  
+
 </h3>
 
 ![image](https://nussknacker.io/documentation/assets/images/nu_designer-87526e47584a5eeb9ce59ad7509d2e7b.png)
 
-Nussknacker lets you design, deploy and monitor real time decision algorithms with easy to use GUI.
+## What is Nussknacker
+
+Nussknacker is a low-code visual tool for domain experts to build, run and monitor real-time decision algorithms instead of implementing them in the code.
+
+In all IT systems, no matter the domain, decisions are made all the time. Which offer should a customer get? Who should receive a marketing message? Does this device need servicing? Is this a fraud?
+
+Algorithms for making such decisions can be developed by programmers. With Nussknacker however, such decision algorithms can be authored and deployed without the need to involve IT.
+
+An essential part of Nussknacker is a visual design tool for decision algorithms. It allows not-so-technical users, like analysts or business people, to author decision logic in an imperative, easy-to-follow and understandable way. Once authored, with a click of a button, scenarios are deployed for execution. And can be changed and redeployed anytime there’s a need.
```

Comment:
Don't you think we should somewhere in this doc switch from "algorithm" to "scenario"?

Reply:
Changed "algorithm node" to "algorithm step". 

The term algorithm is used 15 times in Readme. I am not sure whether changing some of the occurrences of the term "algorithm"  to "Scenario".

Reply:
I started to use term scenario